package.json

```diff
@@ -72,12 +72,17 @@
   },
   "devDependencies": {
     "axios": "^0.18.1",
+    "babel-eslint": "^10.0.1",
```

Comment:
What are the new dev dependencies?

Reply:
Stuff that eslint needs to work with the the eslint/prettier configuration

Reply:
We don't need any of this. Please remove all eslint related dependencies 😊